source/Outfit.cpp

```diff
@@ -289,6 +286,21 @@ void Outfit::Load(const DataNode &node)
 			child.PrintTrace("Skipping unrecognized attribute:");
 	}
 
+	// If no plural name has been defined, append an 's' to the name and use that.
+	// If the name ends in an 's' or 'z', and no plural name has been defined, print a
+	// warning since an explicit plural name is always required in this case.
+	// Unless this outfit definition isn't declared with the `outfit` keyword,
+	// because then this is probably being done in `add attributes` on a ship,
+	// so the name doesn't matter.
+	if(pluralName.empty())
+	{
+		pluralName = name + 's';
+		if((name.back() == 's' || name.back() == 'z') && node.Token(0) == "outfit")
+			node.PrintTrace("Warning: outfit \"" + name +
+					"\" requires an explicit plural name definition, but none is provided. Defaulting to \""
```

Comment:
Since we're doing a node trace, isn't naming the outfit in the warning redundant? We aren't naming the ship in the warning there.

Reply:
Yes